trin-execution/src/storage/evm_db.rs

```diff
@@ -102,6 +106,15 @@ impl EvmDB {
         trie_diff
     }
 
+    pub fn get_newly_created_contract_if_available(&self, code_hash: B256) -> Option<Bytecode> {
```

Comment:
nit: I think that `get_newly_created_contract` is clear enough in this case. Up to you.